src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -691,7 +691,7 @@ SubCmdValidate::execute(const SubCmdOptions& _options) const
   }
 
   //get current performance mode
-  const auto curr_mode = xrt_core::device_query<xrt_core::query::performance_mode>(device);
+  const auto curr_mode = xrt_core::device_query_default<xrt_core::query::performance_mode>(device, 0);
```

Comment:
So if it is not supported, it will show default?

Reply:
Yes, it's how it would show up as default in the ptree as well in the previous query.